pkg/api/pod/util.go

```diff
@@ -386,11 +386,13 @@ func GetValidationOptionsFromPodSpecAndMeta(podSpec, oldPodSpec *api.PodSpec, po
 		AllowNamespacedSysctlsForHostNetAndHostIPC:        false,
 		AllowNonLocalProjectedTokenPath:                   false,
 		AllowImageVolumeSource:                            utilfeature.DefaultFeatureGate.Enabled(features.ImageVolume),
+		AllowRelaxedDNSSearchValidation:                   utilfeature.DefaultFeatureGate.Enabled(features.RelaxedDNSSearchValidation),
```

Comment:
kind of a nit, but why set this if you just override it below?

Reply:
Whoops, good catch, thanks.
Fixed in 1290574ceeba3a7787bb76d4d22a9a6749e6557f